app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,9 @@
   </head>
 
   <body>
+    <% flash.each do |name, msg| %>
+      <%= msg %>
+    <% end %>
     <%= yield %>
```

Comment:
Hi @tsheporamantso
Great work creating flash messages, however, it's always advisable to keep this file as clean as possible, thus kindly create a partial file for the flash message and render it on the application.rb file